.github/workflows/ci.yml

```
@@ -0,0 +1,80 @@
+            npm test
+          else
+            echo "No app/package.json - skipping Node job"
+          fi
+
+  test-rust:
+    name: Rust tests (engine)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.10)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)